coralproject/talk
# [CORL-2535]: include bio reported info in comment card

## What does this PR do?

These changes ensure that when bios are reported, these flags are also displayed with more info when someone clicks `DETAILS` on the comment card.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?
 
Make sure bios are enabled in configuration. Report a comment for its bio with supporting text. Look at the comment card in the reported queue. See that if you click `DETAILS`, you will see a `Bio` category with the user who reported the commenter bio and the supporting text added beneath. See that other flags are still included as expected with details in the comment card too.
 
## How do we deploy this PR?
